test/base.py

```diff
@@ -1,12 +1,28 @@
 import unittest
-
 import jsonschema
+import os, sys
 
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from genson import Schema
 
 class SchemaTestCase(unittest.TestCase):
+
+    def assertGenSchema(self, instance, options, expected):
+        actual = Schema(**options).add_object(instance).to_dict()
+        self.assertSchema(actual, expected)
+        self.assertValidData(instance, actual)
```

Comment:
Since this is part of the user contract, I think I'm okay just putting it inside `assertSchema`.